pkg/exchange/okex/convert_futures.go

```diff
@@ -42,25 +44,28 @@ func toGlobalFuturesPositions(futuresPositions []okexapi.Position) types.Futures
 	for _, futuresPosition := range futuresPositions {
 		isolated := futuresPosition.MgnMode == okexapi.MarginModeIsolated
 		symbol := toGlobalSymbol(futuresPosition.InstId)
-		retFuturesPositions[symbol] = types.FuturesPosition{
-			Isolated:    isolated,
-			AverageCost: futuresPosition.AvgPx,
-			Base:        futuresPosition.Pos,
-			Quote:       futuresPosition.Pos.Mul(futuresPosition.MarkPx),
-			PositionRisk: &types.PositionRisk{
-				Leverage: futuresPosition.Lever,
-			},
-			Symbol: symbol,
+		positionSide := toGlobalPositionSide(okexapi.PosSide(futuresPosition.PosSide))
+
+		posKey := fmt.Sprintf("%s:%s", symbol, positionSide)
```

Comment:
why in this format?

Reply:
Since the position mode involves both one-way and hedge (two-way) positions, in the case of hedge mode, there may be a position record for each direction.